app/org/sagebionetworks/bridge/config/BridgeSpringConfig.java

```diff
@@ -367,6 +367,13 @@ public DynamoIndexHelper uploadHealthCodeRequestedOnIndex(AmazonDynamoDBClient d
         return DynamoIndexHelper.create(DynamoUpload2.class, "healthCode-requestedOn-index", dynamoDBClient, dynamoNamingHelper, dynamoUtils);
     }
     
+    @Bean(name = "healthCodeActivityGuidIndex")
+    @Autowired
+    public DynamoIndexHelper healthCodeActivityGuidIndex(AmazonDynamoDBClient dynamoDBClient, DynamoUtils dynamoUtils,
+            DynamoNamingHelper dynamoNamingHelper) {
+        return DynamoIndexHelper.create(DynamoUpload2.class, "healthCodeActivityGuid-localScheduledOn-index", dynamoDBClient, dynamoNamingHelper, dynamoUtils);
```

Comment:
You're right, I replaced that last thing and didn't change that. This would fail tests. Fixing